src/test/java/org/kainos/ea/integration/JobRoleIntegrationTest.java

```diff
@@ -53,6 +51,15 @@ void getJobRolesShouldReturnListOfJobRoles() {
     }
 
     @Test
+    void deleteJobRoleShouldReturnVoid() {
```

Comment:
this integration test is dependent of role with id 1 exist in DB so is a flakey test. please insert a role first, get the ID of the new role then delete it